stack-ghc-8.2.1.yaml

```diff
@@ -1,6 +1,7 @@
-resolver: nightly-2017-09-01
+resolver: nightly-2017-09-13
```

Comment:
sorry, my rebase comment was triggered by this.

I personally don't like bumping resolvers often (and tend to prefer beginning of the months. But that's just me, we don't need to have any policy on this).

Reply:
Fair enough. I'll switch back.